src/Sylius/Bundle/ApiBundle/Resources/config/services.xml

```diff
@@ -31,6 +31,10 @@
             <argument type="service" id="api_platform.symfony.iri_converter" />
         </service>
 
+        <service id="Sylius\Bundle\ApiBundle\Controller\UploadTaxonImageAction" public="true">
```

Comment:
```suggestion
        <service id="Sylius\Bundle\ApiBundle\Controller\UploadTaxonImageAction">
```
Some time ago we had a discussion about whether services should be public and we came to the conclusion that usually they don't

Reply:
True, but only there when we can, here this action need to be public to be callable by the APIP